README.md

```diff
@@ -324,6 +335,20 @@ Types are exported from `./lib/types/[node/web]/index.d.ts` and should be automa
   });
   ```
 
+- `topUpWithTokens({ tokenAmount, feeMultiplier })` - Tops up the connected wallet with Credits by submitting a payment transaction for the token amount to the Turbo wallet and then submitting that transaction id to Turbo Payment Service for top up processing.
+
+  - The `tokenAmount` is the amount of tokens in the token type's smallest unit value (e.g: Winston for arweave token type) to fund the wallet with.
+  - The `feeMultiplier` (optional) is the multiplier to apply to the reward for the transaction to modify its chances of being mined. Credits will be added to the wallet balance after the transaction is confirmed on the given blockchain. Defaults to 1.0, meaning no multiplier.
+
+  ```typescript
+  const turbo = TurboFactory.authenticated({ signer, token: 'arweave' });
```

Comment:
Kind of a bummer that we can't infer the token from the signer 🤔 Might be necessary to split them for EVM wallet/signer though if we decide to support Ethereum L2s.

Reply:
I think we might be able to do that but I'm was kinda following the same pattern as Irys sdk where they always construct with a token

Reply:
Also -- We cannot always infer token from signer because an ed25519 signer would have no token, but could extend into other tokens. I'll need more research into different signers as we add more types